### PR TITLE
index-employee page

### PR DESCRIPTION
**Pull request by yayasdh**
Screen Shot 2022-08-30 at 13 49 27
Employee Index Page

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -13,6 +13,7 @@
 @import "components/index";
 @import "components/sidebar";
 @import "pages/index";
+@import "components/email";
```
Comment (tuoanhnt95) [inline app/assets/stylesheets/application.scss]: Could you move the "email" and "sidebar" to pages/index.scss?

Comment (yayasdh) [reply app/assets/stylesheets/application.scss]: ok!

**Comment by yayasdh** [comment]
Screen Shot 2022-08-30 at 14 00 20

**Review by tuoanhnt95 — APPROVED** [review]

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (tuoanhnt95) [reply app/assets/stylesheets/application.scss]: Thank you. Will merge now!

**Pull request merged**